src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -235,7 +235,7 @@ public void removeAll() {
             }
         }
 
-        return tokens.iterator();
+        return tokens;
```

Comment:
I don't get it. How null could be possible? the tokens could be empty if nothing added, and then it would be an empty iterable.